src/main/java/views/blackjack/blackjack.ftl.html

```diff
@@ -123,6 +123,43 @@
      });
 
 });
+
+//Function to handle hit requests from the user
+//Checks if the field of the hit button is "Hit"
+//If true then we make a post request to get a new card
+//Grab this new card, string configure it, and append
+//the new card to the users hand
+$("#dealHit").click(function(){
+    val = "";
+    data = "";
+    data = $("div.dealHit").text;
+
+    if(data == "Hit"){
+        var newCard;
+        //Send game data to get card{amount}
+        $.ajax({
+            type: "POST",
+            url: "/val/" + val,
+            data: JSON.stringify(game),
+            success: function(data, status){
+                console.log("Data: " + data + "\nStatus: " + status);
+                //Display game data
+                display(data);
```

Comment:
Going back to what we were talking about. Probably have this call a function like displayCards(game) and put the code for checking the error flags sent back from the model in there as well as the actual display code which in this case would be lines 151-155.